ecl/eclccserver/eclccserver.cpp

```diff
@@ -712,10 +725,12 @@ class EclccServer : public CInterface, implements IThreadFactory, implements IAb
 
 void openLogFile()
 {
+#ifndef _CONTAINERIZED
     StringBuffer logname;
     envGetConfigurationDirectory("log","eclccserver",globals->queryProp("@name"),logname);
     Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(logname.str(), "eclccserver");
```

Comment:
Have we decided that components in containers will never log to a file?

Reply:
I think we have decided that they should not need to by default.

This change was really about seeing what was still needed to make eclcc independent of environment.dll - I expect when we have logging fully sorted it may change again (but is ok for now)